gulp/tasks/webpack.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+var gulp = require('gulp'),
+    gulpUtil = require('gulp-util'),
+    webpack = require('webpack'),
+    WebpackDevServer = require('webpack-dev-server'),
+    gitUtils = require('../util/gitUtils'),
+    config = require('../config.js'),
+    wpConfig = require('../../webpack.config.js'),
+    wpConfigFactory = wpConfig.configFactory,
+    wpConfigDevServer = wpConfig.devServerConfig;
+
+/* Builds the pattern library doc into /pldoc/public.
```

Comment:
I would probably make these JSDoc-like, with a single line summary as the first line. They aren't defining functions per se, but they are defining conceptual blocks.